.github/renovate.json

```diff
@@ -17,14 +17,15 @@
       "matchManagers": ["composer"],
       "groupName": "composer updates"
     },
-    {
-      "matchDepNames": ["php"],
-      "matchFileNames": ["scripts/build/Dockerfile"],
-      "enabled": false
-    },
     {
       "matchManagers": ["dockerfile", "docker-compose"],
       "groupName": "docker updates"
+    },
+    {
+      "description": "PHP, composer and Prestashop versions in Dockerfiles must be freezed",
+      "matchManagers": ["docker"],
```

Comment:
```suggestion
      "matchManagers": ["dockerfile"],
```

Reply:
@gdraynz How do we know which manager to put here ?
I put docker because of the name of the PR https://github.com/alma/alma-installments-prestashop/pull/485

Reply:
We explicitly declare our managers in the `enabledManagers` key so that we know what are the active ones.
But the full list is here: https://docs.renovatebot.com/modules/manager/

Reply:
OK I'm just blind, it is literally written in the file and now I understand the title of the PR is coming from the groupName
Well at least now I know, thanks @gdraynz, and sorry for the stupid question 🤦